app/controllers/data/data.controller.js

```diff
@@ -44,6 +56,7 @@ async function tearDown (_request, h) {
 
 module.exports = {
   exportDb,
+  mockData: mock,
```

Comment:
Do we want to rename our `mock` function to be `mockData` instead of aliasing it in the exports?